[framework] dont hit test for systemnav style if there is no bottom window padding

jonahwilliams opened the pull request:
This means there is no bottom navigation bar, so the hit test is wasted.  (at least, I think so...)

jonahwilliams committed:
[framework] dont hit test for systemnav style if there is no bottom window padding

jonahwilliams committed:
[framework] dont hit test for systemnav style if there is no bottom window padding

jonahwilliams committed:
Merge branch 'master' of github.com:flutter/flutter into benchy_mc_benchface

Commit:
Update view.dart

Commit:
Update view.dart

goderbauer commented on packages/flutter/lib/src/rendering/view.dart:
```diff
@@ -273,22 +273,27 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
-      // bottom drawn pixel is at 1919 position.
-      bounds.bottom - 1.0 - _window.padding.bottom / 2.0,
-    );
     final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top);
```
Would the same logic apply to the system status bar? Can this be skipped if there isn't viewPadding at the top?

jonahwilliams replied:
Good idea, let me check that...

jonahwilliams replied:
can you hide this on iOS or just Android?

jonahwilliams replied:
I'll follow up with this one another time

goderbauer reviewed (approved):
LGTM

jonahwilliams commented:
g3 testing failures are due to binding change, I'll wait til that is resolved then rebase

jonahwilliams commented:
or maybe it is already fixed...

jonahwilliams committed:
Merge branch 'master' of github.com:flutter/flutter into benchy_mc_benchface

jonahwilliams committed:
Merge branch 'benchy_mc_benchface' of github.com:jonahwilliams/flutter into benchy_mc_benchface

jpnurmi committed:
docs: adjust the flutter.material.RawMaterialButton.mouseCursor template (#88470)

MichaelTamm committed:
[flutter_tools] Fix bundle file not found when flavor contains upperc… (#92660)

chingjun committed:
Fix error in type cast. (#97778)

devoncarew committed:
use 'dart pub' instead of calling the pub executable directly (#97749)

use 'dart pub' instead of calling the pub executable directly

justinmc committed:
Text editing shift + tap + drag interaction (#95213)

Supports the desktop text editing interaction of holding shift, tapping the field, and dragging to modify the selection.

christopherfujino committed:
[flutter_tools] increment y instead of m when calling flutter --version on master (#97827)

engine-flutter-autoroll committed:
Roll Engine from 1232caf92f83 to f55d4feba892 (4 revisions) (#97834)

jonahwilliams commented:
So I have failing google3 tests due to the Bindings change, but I already pulled latest. how do I get frob to use a different base commit - do I need to rebase?

goderbauer commented:
I don't think there is a way for frob to do that. I think you just have to wait until that change rolled into google3, unfortunately. @renyou would know for sure.

jmagman committed:
Include -isysroot -arch and -miphoneos-version-min when creating dummy module App.framework (#97689)

engine-flutter-autoroll committed:
Roll Engine from f55d4feba892 to f04ac2a4fdd0 (2 revisions) (#97841)

engine-flutter-autoroll committed:
Roll Engine from f04ac2a4fdd0 to 89df0d0d87dc (2 revisions) (#97847)

dependabot[bot] committed:
Bump ossf/scorecard-action from 1.0.2 to 1.0.3 (#97819)

dependabot[bot] committed:
Bump github/codeql-action from 1.0.26 to 1.0.31 (#97820)

ColdPaleLight committed:
Make the application lifecycle does not affect `SchedulerBinding.scheduleForcedFrame`. (#97468)

engine-flutter-autoroll committed:
Roll Engine from 89df0d0d87dc to 45556e526582 (2 revisions) (#97850)

engine-flutter-autoroll committed:
Roll Engine from 45556e526582 to 5a091b756ea9 (2 revisions) (#97852)

engine-flutter-autoroll committed:
Roll Engine from 5a091b756ea9 to bbe4e9758c7b (1 revision) (#97854)

cachapa committed:
Report progress on Dismissible update callback (#95504)

jonahwilliams committed:
++

jonahwilliams committed:
Merge branch 'benchy_mc_benchface' of github.com:jonahwilliams/flutter into benchy_mc_benchface
